Automattic/themes
# Seedlet: fix keyboard navigation

**Summary**

This PR addresses #2284, implementing the following changes:

- Traps keyboard navigation in the mobile modal menu when its open.
- Removes theme focus styles from wpadmin bar.

![Kapture 2020-07-29 at 15 09 27](https://user-images.githubusercontent.com/5375500/88842486-863e5200-d1ad-11ea-9525-d2edfab9ff24.gif)

**How to test**
- Check out this PR
- Add a Primary menu and a WooCommerce product or two to your cart
- Resize to the mobile viewport
- Use keyboard navigation 
- Verify that the keyboard navigation is trapped in the menu when you open it

I've tested on Safari and Chrome so far.

**To-do**
- [x] Change the tabbing order on mobile so that the menus buttons are reached first via keyboard. The visual order does not follow the page markup, and altering the tabindex doesn't seem like a straightforward solution.
- [x] Test thoroughly.
- [x] Consider refactoring this JS to a class.
